src/mean.jl

```diff
@@ -20,8 +20,7 @@ end
 
 """
     mean(x, design)
-
-Compute the mean and SEM of the survey variable `x`.
+Estimate the population mean of a variable of a simple random sample, and the corresponding standard error.
```

Comment:
Again, I think the previous description was more appropriate. This one is not general.

Reply:
@smishr 
1. Should it be “estimate” or “compute”? 
2. should we be explicit that it's “population mean”?

Reply:
How about:

```suggestion
Estimate the population mean and SEM of survey variable.
```

I want to keep it general (not mention simple random sample) so that we don't have the same docstring for stratified but with one different word. We should have only one documentation, the function does the same thing.

Reply:
What if you do: 
```
? mean(x::Symbol, design::StratifiedSample)
```
You won't get a self-contained documentation for `mean(x::Symbol, design::StratifiedSample)`. 

If we want to add anything general, we should do it above @docs.